### DISCUSSION TITLE
If there are multiple error messages for the same hover location, how will the tooltip be like?

### DISCUSSION THREAD

**Discussion by durianwaffle** [eclipse-langium/langium#1693] [Q&A] [answered]
If there are multiple error messages for the same hover location, how will the tooltip be like? Will it show the 1st error message? Or it will show all of the error messages?

**Comment by msujew** [comment, marked as answer]
It will show all of them.

**Reply by durianwaffle** [reply]
Thanks a lot @msujew

**Reply by durianwaffle** [reply]
Will there be a limit for the number of errors displaying at the same location?

**Reply by msujew** [reply]
I don't think so. There might be a limit introduced by the language client (vscode), but Langium does not enforce a limit.

**Reply by durianwaffle** [reply]
Okay thank you @msujew !
